src/MappedArrays.jl

```diff
@@ -58,7 +58,7 @@ not set them).
 When multiple input arrays are supplied, `M[i] = f(A[i], B[i], C[i]...)`.
 """
 function mappedarray(f, data::AbstractArray)
-    T = typeof(f(testvalue(data)))
+    T = Core.Compiler.return_type(f, (eltype(data),))
```

Comment:
Not inferrable (see the test failure).

Reply:
That's interesting, it's passing for me locally on 1.5.2, I wonder what could be causing that.

Reply:
Could it be that travis is activating some flags to reduce inference?

Reply:
Nevermind, somehow the tests were happening on the wrong branch,